Projects/Features/LyricHighlightingFeature/Sources/ViewControllers/LyricHighlightingViewController.swift

```diff
@@ -45,6 +45,14 @@ public final class LyricHighlightingViewController: UIViewController {
         $0.backgroundColor = .clear
     }
 
+    let emptyLabel = UILabel().then {
```

Comment:
```suggestion
    private let emptyLabel = UILabel().then {
```
여기서만 사용하는거면 private이 맞을것같아요

Reply:
Bind() 로직이 다른 파일로 분리가 돼있어서 프라이빗이면 접근이 안되더라구요.

Reply:
아아 분리해두셨구나 확인했습니당